### PR TITLE
Replace align attribute for P and IMG elements

### PR DESCRIPTION
**Pull request by ruiliuinin**

### CODE REVIEW

File: build/squire-raw.js [hunk 1]
```diff
@@ -478,6 +478,28 @@ function fixContainer ( container, root ) {
...
+                dir = doc.dir.toLowerCase();
+                defaultAlignment = dir === 'rtl' ? 'right' : 'left';
+                if ( child.align ) {
+                    alignment = child.align;
+                    child.removeAttribute( 'align' );
+                } else {
+                    alignment = defaultAlignment;
+                }
+                
+                wrapper.className = ( wrapper.className
```
Comment (akauffmanGG) [inline build/squire-raw.js]: Use the _addAlignClassName function here

File: build/squire-raw.js [hunk 2]
```diff
@@ -4117,6 +4149,23 @@ proto.setHTML = function ( html ) {
     // Fix cursor
     var node = frag;
     while ( node = getNextBlock( node, root ) ) {
+        // Replace the align attribute in P tag by style
+        if ( node.tagName.toUpperCase() === 'P' ) {
```
Comment (akauffmanGG) [inline build/squire-raw.js:4154]: This code is duplicated for both IMG and P elements. Can we just extract this into a separate function? Something like fixAlignmentAttributes.

**Review by akauffmanGG — CHANGES_REQUESTED** [review]
I'm assuming these changes are so that we can get the alignment buttons to show correctly. If so, this seems like overkill to add an alignment class to each block, p, and img.

File: build/squire-raw.js (hunk 2 above)
Comment (ruiliuinin) [reply build/squire-raw.js:4154]: P and IMG are slightly different. IMG elements will be wrapped up in DIV so the new align class is added to div and the align attribute is removed from IMG. P elements don't need to wrapped put in DIV so the new classed in added to P. Added a new function _getAlignment to reuse the code in common between these two.

**Comment by ruiliuinin** [comment]
It needs to have an alignment class to each block, p, and img to make sure the path is changed like changing from 'DIV>P.alignment-left' to  'DIV>P.alignment-right'. If we don't have the alignment class, both paths will be "DIV>P". It won't be able to tell the path changed when the cursor moves between such elements.

**Comment by akauffmanGG** [comment]
Ok. I don't feel great about adding an alignment class everywhere. We should look to see if maybe we could only add it when it's different from the body dir or something like that.

**Pull request merged**